imap_processing/tests/swapi/test_swapi_l2.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from imap_processing import imap_module_directory
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.swapi.l1.swapi_l1 import swapi_l1
+from imap_processing.swapi.l2.swapi_l2 import TIME_PER_BIN, swapi_l2
+
+
+def test_swapi_l2_cdf():
+    """Test housekeeping processing and CDF file creation"""
+    l0_data_path = (
+        f"{imap_module_directory}/tests/swapi/l0_data/"
```

Comment:
I suggest creating a fixture that returns a `Path` which points to the l0 test data path and then using that in any test function referencing that path. e.g...
```
@pytest.fixture
def swapi_l0_test_data_path():
    return imap_module_directory / "tests" / "swapi" / "l0_data"

def test_that_uses_swapi_l0_data(swapi_l0_test_data_path):
    test_file_path = swapi_l0_test_data_path / "this_l0_file.pkts"